lib/geojsonLayerController.js

```diff
@@ -51,6 +52,8 @@ function buildGeojsonLayerOptions(options) {
  * @return {object}
  */
 function buildConfigProcessingOptions(layer, config) {
+  var cluster = (layer.styles && layer.styles.icon) ? layer.styles.icon.cluster : null
```

Comment:
Doesn't this controller have the nevernull module in play already? Could possibly use that to prevent all the existence checks